hr_attendance_reason/models/hr_attendance_reason.py

```diff
@@ -10,12 +9,16 @@ class HrAttendanceReason(models.Model):
     _name = "hr.attendance.reason"
     _description = "Attendance Reason"
 
+    _sql_constraints = [
+        ('unique_code', 'UNIQUE(code)', 'Code must be unique')
+    ]
+
     name = fields.Char(
         String='Reason',
         help='Specifies the reason leaving soon or arriving late',
         required=True, index=True)
     code = fields.Char('Reason Code')
     action_type = fields.Selection(
-        [('sign_in', 'Sign in'),
```

Comment:
Do not change the name without a migration script please.
You can make the script or keep the old name.

Reply:
Okay, I will keel sign in and sign out then.